README.md

```diff
@@ -164,6 +164,41 @@ A default `getA11yStatusMessage` function is provided that will check `resultCou
 
 Called when the user selects an item
 
+#### onStateChange
+
+> `function({highlightedIndex, inputValue, isOpen, selectedValue})` | not required, no useful default
+
+This function is called anytime there's a state change internally. This can be
+useful if you're using the component as a "controlled" component (passing the
+state in rather than letting the component control the state itself).
```

Comment:
I think this is worded in a confusing way. ‘The component’ in this case is the Downshift component? But sounds like it could be talking about your component too.

Maybe:
“This function is called anytime the internal state changes. This can be useful if you're using Downshift as a "controlled" component, where you manage the state (e.g. `isOpen`, `selectedValue`, `highlightedIndex`, etc) and then pass it as props, rather than letting Downshift control all its state itself.”